behave_test_automation/pah_behave/features/steps/logout_steps.py

```diff
@@ -0,0 +1,32 @@
+from behave import when, given, then
+
+
+@given('User is on drive page')
+def login_successful(context):
+    context.login_page.login_to_pah_website()
+    context.login_page.login_successful()
+    assert context.driver.execute_script("return window.localStorage.jwt") is not None
```

Comment:
calls to `execute_script()` should be hidden behind utility function.

Reason behind it is that you don't care _how_ the actual check for login works internally. All you care about in terms of code that has to be written for this step function called `login_successful()` is `assert user_is_logged_in()`.

The `user_is_logged_in()` then checks it via `execute_script()`. But that's the magic - it _could_ check it via some call to API, or maybe by checking if `Logout` button is present. No matter the way, it does it's job via some hidden unknown shenanigans, which, in this case is a call to `execute_script()`. But that does not matter to us at the step implementation level.

Reply:
You're right, moved all asserts from step implementation to methods.